src/main/resources/org/jenkinsci/plugins/p4/populate/AutoCleanImpl/config.jelly

```diff
@@ -11,6 +11,10 @@
 	<f:entry field="modtime">
 		<f:checkbox title="${%Sync with MODTIME for consistency check}" default="false"/>
 	</f:entry>
+
+	<f:entry field="quiet">
+		<f:checkbox title="${%Suppress Perforce generated info messages}" default="false"/>
```

Comment:
Perhaps a default of 'true'?
The current release 1.2.2 is very quiet.  It is only the recent changes in Main that added the Perforce output to the console log.